Lib/tokenize.py

```diff
@@ -0,0 +1,803 @@
+"""Tokenization help for Python programs.
+
+tokenize(readline) is a generator that breaks a stream of bytes into
```

Comment:
This module we have as a rust module. So we would not require this python version as well.